ipaddr.py

```diff
@@ -30,7 +30,7 @@ def _str2int(ip):
 
 def _int2str(num):
     bytes = struct.unpack("BBBB", struct.pack("!L", num))
-    return string.join(list(map(str, bytes)), '.')
+    return '.'.join(list(map(str, bytes)))
```

Comment:
the `list` usage is superfluous but doesn't change behavior. `.join(...)` takes an iterable, so a `map` object acts the same as a `list` object in this case.

Reply:
Ah ok, thanks for that, FWIW, I'm pretty sure that the `list(map(...` was added by `2to3`.